package.json

```diff
@@ -4,7 +4,7 @@
   "main": "index.js",
   "scripts": {
     "commit": "git-cz",
-    "check-coverage": "istanbul check-coverage --statements 96 --branches 81 --functions 87 --lines 96",
+    "check-coverage": "istanbul check-coverage --statements 96 --branches 84 --functions 85 --lines 96",
```

Comment:
Actually, do you mind removing the `getTypes` function. I really should add linting to this repo. :-)